lib/control/control-commands.c

```diff
@@ -178,9 +178,10 @@ process_pwd(GString *command, gpointer user_data)
         {
           if (!secret_storage_store_string(cmds[2],cmds[3]))
             {
-              g_string_assign(result,"Error during store process in secret store\n");
+              g_string_assign(result,"Error while saving credentials\n");
             }
         }
+      g_string_assign(result, "OK");
```

Comment:
We need to replace OK with "Credentials stored". OK is misleading: users might think the password was correct, but what we  report here is the store was successful.

Reply:
Fixed.